OrbitGl/FunctionsDataView.cpp

```diff
@@ -15,6 +15,9 @@ using orbit_client_protos::FunctionInfo;
 
 FunctionsDataView::FunctionsDataView() : DataView(DataViewType::kFunctions) {}
 
+const std::string FunctionsDataView::kSelectedFunctionString = "✓";
+const std::string FunctionsDataView::kNonSelectedFunctionString = "";
```

Comment:
Nit: Not 100% sure, but "non selected" sounds wrong to me. "Unselected", maybe?

Reply:
The reasoning was that "unselected" feels like a functions was selected and then deselected, but I don't like "non selected" either, so I changed it.